MapboxNavigation/RouteVoiceController.swift

```diff
@@ -207,3 +211,8 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         speechSynth.speak(utterance)
     }
 }
+
+@objc public protocol VoiceControllerDelegate {
+    @objc optional func voiceController(_ voiceController: RouteVoiceController, spokenInstructionsDidFailWith error: Error)
```

Comment:
The Objective-C signature of this method is now `-voiceController:spokenInstructionsDidFailWith:`. Objective-C selector pieces should end in nouns, not prepositions like in Swift. Use `@objc(…)` to customize the Objective-C selector.

Reply:
This method still needs a proper Objective-C signature.